packages/paypal-commerce-integration/src/paypal-commerce-alternative-methods/paypal-commerce-ratepay-payment-strategy.ts

```diff
@@ -193,6 +217,7 @@ export default class PaypalCommerceRatepayPaymentStrategy implements PaymentStra
 
     private handleError(error: unknown): void {
         const { onError } = this.paypalcommerceratepay || {};
+        this.resetPollingMechanism();
```

Comment:
Should we reset polling mechanism each time when we get any error?

Reply:
Yes, the same we did in` paypal-commerce-alternative-methods-payment-strategy`